apps/charterafrica/src/components/ErrorPage/ErrorPage.js

```diff
@@ -0,0 +1,84 @@
+import RichTypography from "@/commons-ui/core/RichTypography";
+import Section from "@/commons-ui/core/Section";
+import { Link } from "@commons-ui/next";
+import { Button } from "@mui/material";
+import PropTypes from "prop-types";
+import React from "react";
+
+import { secondary, neutral } from "@/charterafrica/colors";
+import BackgroundBox from "@/charterafrica/components/BackgroundBox";
+import RichText from "@/charterafrica/components/RichText";
+
+const ErrorPage = React.forwardRef(function Error(props, ref) {
+  const {
+    statusCode,
+    description = [],
+    title = "",
+    createdAt,
+    updatedAt,
+    blockType,
+    action,
+    ...other
+  } = props;
+  return (
+    <BackgroundBox
+      {...other}
```

Comment:
```suggestion
```

Avoid this. For example, if you've received props with other params like id, etc., they'll all end up added to DOM.